gempak/ush/gdas_meta_na.sh

```diff
@@ -16,13 +16,14 @@ if [[ ! -L "${COMIN}" ]]; then
     ${NLN} "${COMIN_ATMOS_GEMPAK_1p00}" "${COMIN}"
 fi
 
-if [[ "${envir}" == "para" ]] ; then
-   export m_title="GDASP"
+if [[ "${envir}" == "para" ]]; then
+    export m_title="GDASP"
 else
-   export m_title="GDAS"
+    export m_title="GDAS"
 fi
 
-export pgm=gdplot2_nc; prep_step
+export pgm=gdplot2_nc
+prep_step
```

Comment:
This line may need to become "`source prep_step`", consistent with other `gempak/ush` scripts.

Reply:
Thank you for this catch @ChristopherHill-NOAA.  This PR was only to bring the GFS v17 branch up to date with develop.  This issue also exists in develop, so this should be handled in a separate PR to develop.